src/business-logic/services/got-http-service.ts

```diff
@@ -0,0 +1,10 @@
+import {HttpRequestParameters, HttpService} from './interfaces/http-service'
+import got, { Response } from 'got'
+
+export class GotHttpService implements HttpService {
+
+  public async post(httpParameters: HttpRequestParameters): Promise<Response<string>> {
```

Comment:
It seems a bit overkill to have an interface for this. We can just take two parameters. The URL and the body.

Reply:
It has been removed and replaced with `url` and `body`.